docs/source/process.rst

```diff
@@ -115,33 +79,29 @@ By default, echopype uses the following for calibration:
 
 - EK60: Environmental parameters saved with the data files
 
-- AZFP: salinity = 29.6 PSU, pressure = 60 dbar,
+- EK80: Environmental parameters saved with the data files
+
+- AZFP: Salinity and pressure provided by the user,
   and temperature recorded at the instrument
 
 These parameters should be overwritten when they differ from the actual
 environmental condition during data collection.
-To update these parameters, simply do the following *before*
-calling ``ed.calibrate()``:
+To update these parameters, simply pass in the environmental parameters
+as a dictionary while calling ``ep.calibrate.compute_Sv()`` like so:
 
 .. code-block:: python
 
-   ed.temperature = 8   # temperature in degree Celsius
-   ed.salinity = 30     # salinity in PSU
-   ed.pressure = 50     # pressure in dbar
-   ed.recalculate_environment()  # recalculate related parameters
-
-This will trigger recalculation of all related parameters,
-including sound speed, seawater absorption, thickness of each sonar
-sample, and range. The updated values can be retrieved with:
+   environment = {
+       'temperature': 8,   # temperature in degree Celsius
+       'salinity': 30,      # salinity in PSU
+       'pressure': 50,     # pressure in dbar
+   }
+   ep.calibrate.compute_Sv(env_params=environment)
```

Comment:
This looks incomplete. Compare it to the example above: `Sv = ep.calibrate.compute_Sv(echodata)`